### PR TITLE
Use PAT

### PR DESCRIPTION
**Pull request by thomasyu888**

### CODE REVIEW

**Comment by BrunoGrandePhD** [comment]
Did you update the PAT? It still seems to be failing due to `SynapseAuthenticationError`.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -33,8 +33,7 @@ jobs:
           OUTPUT_FILE=/tmp/.synapseConfig
           cat > "$OUTPUT_FILE" << EOM
           [authentication]
-          username = "${{secrets.SYNAPSE_USERNAME }}"
-          apikey = "${{secrets.SYNAPSE_API_KEY }}"
+          authtoken = "${{secrets.SYNAPSE_PAT }}"
```
Comment (thomasyu888) [inline .github/workflows/ci.yaml]:
```suggestion
          authtoken = "${{secrets.SYNAPSE_PAT}}"
```

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -33,8 +33,7 @@ jobs:
           OUTPUT_FILE=/tmp/.synapseConfig
           cat > "$OUTPUT_FILE" << EOM
           [authentication]
-          username = "${{secrets.SYNAPSE_USERNAME }}"
-          apikey = "${{secrets.SYNAPSE_API_KEY }}"
+          authtoken = "${{secrets.SYNAPSE_PAT}}"
```
Comment (thomasyu888) [inline .github/workflows/ci.yaml]:
```suggestion
          authtoken = ${{secrets.SYNAPSE_PAT}}
```

**Comment by thomasyu888** [comment]
@BrunoGrandePhD Turns out my secret key had an extra space in it... Should be good now.

**Review by tthyer — APPROVED** [review]

**Pull request merged**